### ISSUE TITLE
FluidBufLoudness hopSize default

### ISSUE THREAD

**Issue by tedmoore** [flucoma/flucoma-sc#14]
Greetings FluCoMa,

I just noticed that objects FluidBufSpectralShape, FluidBufPitch, FluidBufMFCC, and FluidBufMelBands all have a hopSize default of -1 (which I understand to be windowSize / 2), however FluidBufLoudness has a hopSize default of 512, which is the default windowSize of 1024 / 2, but is inconsistent with the other objects.

Thanks!

**Comment by tedmoore** [comment]
Same with FluidLoudness it looks like.

**Comment by tremblap** [comment]
Indeed, but there was a reason IIRC. I think that is because it did not use the same OLA mechanism under the hood. Changing this property is a good idea but not as trivial as it seems since it is done at the lowest level, and the other implementations (everything FFT) allows to modulate the windowSize and if hopSize is -1 it will modulate accordingly (keeping an overlap of 2)

This is a good example of the other behaviour that we would need to emulate:
```supercollider
{FluidSTFTPass.ar(SinOsc.ar(110,mul: 0.1),windowSize: LFTri.kr(0.1,mul: 2000,add: 2064).poll)}.play
```

**Comment by tedmoore** [comment]
@weefuzzy is this feasible for milestone beta5?

**Comment by weefuzzy** [comment]
At the C++ level it's deep, and certainly not within beta5 timeframe or anything like it. If you just wanted it to be different in SC, then the class could be altered to simulate the behaviour, but this would make SC diverge

**Comment by tremblap** [comment]
We should refrain to do SC-only interface change. Let's park it until we can do it in C++
